tests/env/express/tests/client.test.js

```diff
@@ -4,10 +4,9 @@ describe('MeiliSearch JS Browser test', () => {
   })
 
   it('Should have generated a meilisearch client and displayed', async () => {
-    jest.setTimeout(100 * 1000)
     await page.waitForSelector("#indexes")
     await expect(
       page.content()
       ).resolves.toMatch('createdIndexTest')
-  }, 10000)
+  }, 30000)
```

Comment:
Could you remind me the difference between this `30000` and the one in the `jest-setup.js` file please? (sorry)